src/components/Header/index.tsx

```diff
@@ -4,28 +4,59 @@ import Button from '@components/Button'
 import ProfileName from '@components/ProfileName'
 import Image from 'next/image'
 import styled from 'styled-components'
-
-export type HeaderProps = {
-  isUserLoggedIn: boolean
-}
+import SignInModal from '@components/Form/SignInModal'
+import { useState } from 'react'
+import { useAuth } from '@context/auth'
+import { useMe, UseMeOutput } from '@hooks/queries/useMe'
```

Comment:
Esse import poderia ser mais simples, do tipo `import { useMe, UseMeOutput } from "@hooks/queries"` ou apenas "@hooks"

Reply:
I fix this in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/35/commits/b3f897da8729940a69158281e93fdf538178df36)